app/admin/RTMediaAdmin.php

```diff
@@ -1,34 +1,57 @@
 <?php
-
 /**
- * Description of RTMediaAdmin
+ * Description of RTMediaAdmin.
  *
  * @package    RTMedia
  * @subpackage Admin
- *
  */
+
 if ( ! class_exists( 'RTMediaAdmin' ) ) {
 
 	/**
 	 * RTMediaAdmin class.
 	 */
 	class RTMediaAdmin {
-
+                /**
+                 * Member variables
```

Comment:
@pranali333 Please add description about `$rtmedia_upgrade` property. Need to add property description at other places.